00-programming-fundamentals/project-submissions-without-git.md

```diff
@@ -0,0 +1,96 @@
+
+### Fork the repo correctly
+
+First, you'll want to fork the repo from the Ada-CX organization.
+
+1. To do this, first you'll want to visit the Github.com page of this original repo. Typically, this will be the original project link you received when you started the project. **`AdaGold` _should definitely not_ be in the URL or header of GitHub**
+
+1. Once you're on the right page, find the "Fork" button towards the top right corner
+![Git Fork Step](./images/project-submissions-without-git/find-fork-button.png)
+
+1. This should automatically take you to a page that is your fork: a copy of the repo, but under your account. Verify that you are on the correct page by ensuring that `Ada-CX` nor `AdaGold` are not the repo title, and in fact, your username is! (Check the page title and the browser URL)
+![Your Fork](./images/project-submissions-without-git/verify-your-fork.png)
+
+If you weren't transported to your copy of the repo, you can find a list of your repos through Github.com and finding your account page.
+
+### In your repo, add your code
+
+The next step is to add a new file, and the code to put in that new file, to represent your project. All of this work should be done on your own personal fork.
+
+1. Create a new file on your personal fork by finding the button that says "Create new file"
+![Create file](./images/project-submissions-without-git/create-file.png)
+
+1. This should open up a page in which you put in the details of this new file. Give this file a name with a file extension
+
+1. From your text editor, select all of your code (`cmd+a`), copy it, and then paste it into the large textbox. This represents the contents of the file that you are submitting.
+![new file contents](./images/project-submissions-without-git/new-file.png)
+
+1. At the bottom of the page, you should see details about how to "commit the new file." Use all of the default form values here, and press that fancy "Commit new file" green button!
+![commit](./images/project-submissions-without-git/commit-file.png)
+
+### Open and Create a Pull Request
+
+Almost there -- to actually finish project submission, you'll need to open and create a pull request.
+
+1. Finishing the commit should bring you back to your own fork's page, but if not, make sure you're back there. Then, find the button that says "New pull request" and click it
+![New PR](./images/project-submissions-without-git/new-pr.png)
+
+1. This should open up a new page. PRs are ways in GitHub that we can compare code changes between repos easily. You need to change these settings so that the PR compares the right two things: You'll need to configure it so that it says your username on the right under "head fork," and your cohort org name on the left under "base fork." **Without these settings configured correctly, your project will not submit!** Github tries to be helpful by predicting what you will compare, but it's not very accurate sometimes.
```

Comment:
Since the process doesn't technically break when they configure it poorly, I would change the wording in the bolded section to:

**Without these settings configured correctly, we won't see that your project has been submitted!**